todo/users/routes.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import APIRouter, Request
+from fastapi_users.authentication import JWTAuthentication
+from fastapi_users import FastAPIUsers
+
+from ..config import settings
+from .models import (
+    User,
+    UserCreate,
+    UserUpdate,
+    UserDB,
+    user_db
+)
+
+jwt_authentication = JWTAuthentication(
+    secret=settings.SECRET_KEY,
+    lifetime_seconds=settings.TOKEN_EXPIRES_IN_SECONDS,
+    tokenUrl='auth/jwt/login'
```

Comment:
Does it really use `snake_case` and then suddenly `dromedaryCase` for its arguments?  Looks like a formula for making a mistake.

Reply:
Yeah, it's not very pythonic and uncomfortable to look at. But it's like that because it is using the same name as in the OpenAPI spec 🤷‍♂️

I will focus more on Spring Boot, and try to recreate this using it, so we can all be ready 👍